packages/shared/src/lib/featureManagement.ts

```diff
@@ -16,11 +16,8 @@ export class Feature<T extends JSONValue> {
 const feature = {
   feedVersion: new Feature('feed_version', 15),
   lowImps: new Feature('feed_low_imps'),
-  bookmarkOnCard: new Feature('bookmark_on_card', false),
-  readingStreaks: new Feature(
-    'reading_streaks',
-    ReadingStreaksExperiment.Control,
-  ),
+  bookmarkOnCard: new Feature('bookmark_on_card', true),
```

Comment:
I'd rather we keep this as is until we cleanup

Reply:
Ah, my bad. Was testing before merging. Accidentally pushed.